src/cart/README.env

```diff
@@ -112,25 +112,33 @@ This file lists the environment variables used in CaRT.
    Set it as a port number, when creating NA classes it will use the consecutive
    port numbers start from it.
    When it is not set, then will use random port number.
+   Supports a comma separated list of ports, similar to D_INTERFACE handling.
 
  . D_PORT_AUTO_ADJUST
    Automatically adjusts provided port (via D_PORT) to be within a valid range for
    a given provider. Currently only implemented for CXI to adjust port to be within
    0-511 range.
 
  . D_INTERFACE (Deprecated: OFI_INTERFACE)
-   Set it as the network device name to be used for OFI communication, for
+   Set it as the network device name to be used for network communication, for
    example "eth0", "ib0" or "ens33" etc.
+   Supports a comma separated list of interfaces, for example "eth0,ib0,ens33".
+   If multiple interfaces are specified then CaRT is initialized on all of them,
+   allowing allocation of CaRT contexts on them using crt_context_create_on_iface_idx() API.
+   If no interface is specified, the network layer will select one automatically
+   during context creation (crt_context_create() API)
```

Comment:
since we do not use D_INTERFACE in DAOS client, where can this be provided in cart init?

Reply:
most D_* params such as D_INTERFACE, D_PROVIDER, D_PORT etc.. we have a corresponding crt_init_options_t setting that daos is using today when initing cart with crt_init_opt().

In this case for the interface it would be crt_init_options_t::cio_interface field, just like it is today.

Reply:
I wonder, should we be using D_INTERFACE/etc. in the client? I see we are still using the old OFI_INTERFACE/etc. there. Seems like either the client should be setting both or Cart detecting both.

Reply:
@kjacque when updating the code we should be switching to D_* variant of envariables and away from old OFI_* ones. Currently cart still supports both, with D_* variants (e.g. D_INTERFACE) taking a precedence if set. OFI_* versions have been marked as deprecated for a while now.

Reply:
I created a ticket for this. It's a bit of a tangent from this changeset, but IMO we should begin this transition in libdaos. https://daosio.atlassian.net/browse/DAOS-15401